apps/src/applab/applab.js

```diff
@@ -60,10 +60,15 @@ var StepType = {
   OUT:  3,
 };
 
+var stdAppWidth = 400;
+var defaultAppHeight = 400;
```

Comment:
Is there a semantic difference between standard and default? I would prefer to have them both named with the same prefix, or to have the difference (or identicalness) noted here in comments,